tools/osbuilder/node-builder/azure-linux/uvm_build.sh

```diff
@@ -38,10 +39,18 @@ fi
 
 pushd "${repo_dir}"
 
-echo "Building rootfs and including pre-built agent binary"
+LOCAL_IMAGE_NAME="${IMG_FILE_NAME}"
+IGVM_DEBUG="false"
+
+if [ "${MODE}" == "debug" ]; then
+	LOCAL_IMAGE_NAME="${IMG_DBG_FILE_NAME}"
+	IGVM_DEBUG="true"
+fi
```

Comment:
nit: If possible, I would put all declarations in if branches, ie. `if x { a = ...} else { a = ... }`